key-import-export/rsa/export_app/export_raw_key_from_apc_with_rsa_wrap.py

```diff
@@ -0,0 +1,241 @@
+'''
+Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+Permission is hereby granted, free of charge, to any person obtaining a copy of this
+software and associated documentation files (the "Software"), to deal in the Software
+without restriction, including without limitation the rights to use, copy, modify,
+merge, publish, distribute, sublicense, and/or sell copies of the Software, and to
+permit persons to whom the Software is furnished to do so.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+The following api calls may be subject to https://aws.amazon.com/service-terms/ section 2 - Beta & Previews
+
+Usage - python3 POC_ExportKey_RSAWrapUsingAPI.py
+'''
+
+import requests
+import logging
+import http
+from datetime import datetime
+from datetime import datetime
+
+import botocore.session
+from botocore.auth import SigV4Auth
+from botocore.awsrequest import AWSRequest
+import requests
+import json
+import argparse
+import base64
+import binascii
+import boto3
+from cryptography import x509
+import binascii
+from cryptography.hazmat.primitives import serialization
+import base64
+from Crypto.Hash import SHA256
+from Crypto.Cipher import DES3
+from Crypto.PublicKey import RSA
+from Crypto.Cipher import PKCS1_OAEP
+from cryptography import x509
+from cryptography.x509.oid import NameOID
+from cryptography.hazmat.primitives.asymmetric import rsa, padding
+from cryptography.hazmat.primitives import hashes, serialization
+import binascii
+from datetime import timedelta
+
+
+regionName = 'us-east-1'
```

Comment:
I think the following 5 lines aren't needed anymore?

Reply:
done